sac2c-variables.cmake

```diff
@@ -6,19 +6,34 @@
 #    * TARGET has been set to the target name we compile sac modules for.
 
 
+IF (NOT CMAKE_COMMON_DIR)
+    SET (CMAKE_COMMON_DIR "cmake-common")
+ENDIF ()
+INCLUDE ("${CMAKE_SOURCE_DIR}/${CMAKE_COMMON_DIR}/check-sac2c-feature-support.cmake") # for CHECK_SAC2C_SUPPORT_FLAG
 # Sanity check.  Check that we have a variable with sac2c executable.
 IF (NOT SAC2C_EXEC)
     MESSAGE (FATAL_ERROR "The SAC2C_EXEC variable is not set. (include check-sac2c.cmake?)")
 ENDIF ()
 
+# Sanity check.  Check that we have set SAC2C_FLAGS
+IF (NOT SAC2C_FLAGS)
+    MESSAGE (FATAL_ERROR "The SAC2C_FLAGS variable is not set. (include check-sac2c.cmake?)")
+ENDIF ()
+
 # Sanity check. Check that TARGET is defined.
 IF (NOT TARGET)
     MESSAGE (FATAL_ERROR "The TARGET variable is not set, which sac2c target shall we use?")
 ENDIF ()
 
 # Create local copy of the SAC2C flags
-SET (SAC2C_T ${SAC2C_EXEC} ${SAC2C_CPP_INC} -target ${TARGET})
+SET (SAC2C_T ${SAC2C_EXEC} ${SAC2C_FLAGS} ${SAC2C_CPP_INC} -target ${TARGET})
 SET (SAC2C ${SAC2C_T} -Xc "\"${SAC2C_EXTRA_INC}\"" -Xtc "\"${SAC2C_EXTRA_INC}\"")
```

Comment:
this line should be removed, the conditional below sets `SAC2C`...

Reply:
Doing it now.